### PR TITLE
Clarify inline docs, web/web.ex

### PR DESCRIPTION
**Pull request by nicholaswyoung**
I think this edit clarifies the purpose of `web/web.ex`.

### CODE REVIEW

File: installer/templates/new/web/web.ex [hunk 1]
```diff
@@ -1,6 +1,6 @@
 defmodule <%= application_module %>.Web do
   @moduledoc """
-  A module that keeps using definitions for controllers,
+  A module containing the base definitions for controllers,
```
Comment (chrismccord) [inline installer/templates/new/web/web.ex]: How about `A module defining `__using__` hooks for controllers.` ?

Comment (nicholaswyoung) [inline installer/templates/new/web/web.ex]: Even better.

**Comment by nicholaswyoung** [comment]
Rebased this branch against master, and changed the line @chrismccord and I discussed above.

**Comment by chrismccord** [comment]
<3 <3 <3

**Pull request merged**